### DISCUSSION TITLE
Social Research Findings

### DISCUSSION THREAD

**Discussion by UX-P** [bisq-network/bisq2#181] [Ideas]
I will post ideation documents and findings in this discussion to help with UX, design and development considerations. Feel free to use this discussion as a brainstorm for ideas.

**Comment by UX-P** [comment]
<img width="1060" alt="brainstorm" src="https://user-images.githubusercontent.com/92308399/160852668-7cb00063-7322-475f-a539-f275269ae5c0.png">

**Comment by UX-P** [comment]
<img width="905" alt="Screenshot 2022-03-30 at 14 45 15" src="https://user-images.githubusercontent.com/92308399/160852754-4252f55d-e182-41d2-bd70-dbee632486c8.png">

**Comment by UX-P** [comment]
[Satoshi Square Affinity Map](https://miro.com/app/board/uXjVOBHnMGg=/)

**Comment by UX-P** [comment]
Example of overview screens using the card elements (sourced online):
![overview 3](https://user-images.githubusercontent.com/92308399/164322021-1d7e0f44-c407-424a-8554-0e50d6429443.png)

**Comment by UX-P** [comment]
Here are two main personas for the reputable protocol:

<img width="850" alt="Screenshot 2022-05-27 at 19 37 13" src="https://user-images.githubusercontent.com/92308399/170770870-5421c683-ce22-44eb-870d-788063146a47.png">

<img width="864" alt="Screenshot 2022-05-27 at 19 38 55" src="https://user-images.githubusercontent.com/92308399/170771112-ef578b63-4a16-4565-854d-6eed446d4644.png">

**Comment by HenrikJannsen** [comment]
@UX-P Finally after long work we have release Bisq 2. https://bisq.network/blog/bisq-2-now-in-beta/
Thanks for all your great inputs!!!!

**Reply by UX-P** [reply]
@HenrikJannsen That’s excellent! Love the presentation, really exciting features! I haven’t had a moment to be active for a while. Now that I am more settled, I would love to continue contributing to the project. 
I can gather feedback from users based on their experiences during the months since the Bisq2 launch to confirm it is working as expected. Feedback can go towards making improvements and helping with design features for upcoming releases.  How does that sound?

**Reply by HenrikJannsen** [reply]
Hi @UX-P great to hear back from you! Are you on Matrix as well? We have a Bisq2 room there (https://matrix.to/#/#bisq.v2.dev:bitcoin.kyoto) which is best suited for discussions.

**Reply by UX-P** [reply]
Great! I'll join
